matchbook-sdk-core/src/main/java/com/matchbook/sdk/core/disruptor/messages/UserMessage.java

```diff
@@ -0,0 +1,35 @@
+package com.matchbook.sdk.core.disruptor.messages;
+
+import com.matchbook.sdk.core.model.dataobjects.user.Balance;
+
+public class UserMessage extends AbstractDisruptorMessage {
+
+    private static final long serialVersionUID = -8425010391202208013L;
+
+    private Balance balance;
+    private com.matchbook.sdk.core.clients.rest.dtos.user.Balance balanceDTO;
```

Comment:
Should we rename DTO's using `DTO` suffix?

Reply:
i don't think so,  i have an idea to make REST API a separate library